weave/execute.py

```diff
@@ -455,6 +455,9 @@ def _tag_safe_deref(ref):
     return res
 
 
+client_cache_key_op_allow_list = ["gqlroot-wbgqlquery"]
```

Comment:
This seems like is should be in op_policy.py. You can just add gqlroot-wbgqlquery to CACHE_OP_NAMES and then use this client key if an op we're trying to cache is impure.

Reply:
ok - will update